spec/libsass/nested_interpolation.hrx

```diff
@@ -0,0 +1,14 @@
+<===> input.scss
```

Comment:
Please update this spec to follow [the style guide](https://github.com/sass/sass-spec/blob/main/STYLE_GUIDE.md), including putting it in a directory that matches the part of the language it tests.